### PR TITLE
update translations

### PR DESCRIPTION
**Pull request by r10s**
just a `./tools/t-dance pull`, last one was about 3w ago

### CODE REVIEW

File: it/gdpr-website.md [hunk 1]
```diff
@@ -1,64 +1,62 @@
----
-title: Privacy policy website
+titolo: Privacy del sito web
```
Comment (ghost) [inline it/gdpr-website.md]: Can you do a pull again? I changed that in transifex.

**Review by ghost — CHANGES_REQUESTED** [review]
Apart from that it looks good :) it doesn't yet pull yaml though, so I'll do this again after merging #283 .

File: it/gdpr-website.md (hunk 1 above)
Comment (r10s) [reply it/gdpr-website.md]: thanks, good catch!
maybe we should add a simple check to the t-dance script.

**Review by ghost — APPROVED** [review]
Thanks!

**Pull request merged**